Update defender-for-devops-introduction.md (#103623)
Azure/azure-content: articles/defender-for-cloud/defender-for-devops-introduction.md

Defender for DevOps helps unify, strengthen and manage multi-pipeline DevOps security. 

## Availability
  > [!Note] 

>   > During preview, the GitHub connection will have a max limit of 2,000 repositories onboarded to Microsoft Defender for Cloud. If you have over 2,000 repositories, only the first 2,000 repositories sorted alphabetically will be onboarded.  

Reviewer: ```suggestion
  > During the preview, the maximum number of repositories that can be onboarded to Microsoft Defender for Cloud is 2,000. If you try to connect more than 2,000 repositories, only the first 2,000 repositories, sorted alphabetically, will be onboarded.  
```
